.github/workflows/actionlint.yaml

```diff
@@ -65,4 +65,5 @@ jobs:
         echo "actionlint found ${{ steps.actionlint.outputs.total-errors }} errors"
         echo "actionlint checked ${{ steps.actionlint.outputs.total-files }} files"
         echo "actionlint cache used: ${{ steps.actionlint.outputs.cache-hit }}"
-        exit ${{ steps.actionlint.outputs.exit-code }}
+        echo "${{ steps.actionlint.outputs.exit-code }}" >&2
+        exit 1
```

Comment:
This error is shellcheck error "[SC2242: Can only exit with status 0-255. Other data should be written to stdout/stderr.](https://github.com/koalaman/shellcheck/wiki/SC2242)" And then I used `exit 1` because this actionlint summary says if != 0 and do echo only when actionlint action failed.